src/Microsoft.ML.Transforms/NAReplaceTransform.cs

```diff
@@ -27,13 +27,14 @@
 
 namespace Microsoft.ML.Runtime.Data
 {
-    /// <summary>
-    /// This transform can transform either scalars or vectors (both fixed and variable size),
-    /// creating output columns that are identical to the input columns except for replacing NA values
-    /// with either the default value, user input, or imputed values (min/max/mean are currently supported).
-    /// Imputation modes are supported for vectors both by slot and across all slots.
-    /// </summary>
-    /// REVIEW: May make sense to implement the transform template interface.
+    // <summary>
+    // This transform can transform either scalars or vectors (both fixed and variable size),
+    // creating output columns that are identical to the input columns except for replacing NA values
+    // with either the default value, user input, or imputed values (min/max/mean are currently supported).
+    // Imputation modes are supported for vectors both by slot and across all slots.
+    // </summary>
+    // REVIEW: May make sense to implement the transform template interface.
+    /// <include file='doc.xml' path='doc/members/member[@name="NAReplace"]/*' />
```

Comment:
I think the XML comments *have* to be tripple `///` indented.

I think this `// REVIEW` comment might be messing up the XML parsing. Maybe it should go somewhere else (or even away, it's not adding much info). My suspicion is that this comment isn't actually being compiled into the docs.

Reply:
Oh... wait were you trying to make it no longer an XML doc comment?

In that case you might want to take our those `<summary>` tags.

---
In reply to: [202461616](https://github.com/dotnet/machinelearning/pull/529#discussion_r202461616) [](ancestors = 202461616)